var/spack/repos/builtin/packages/py-xtb/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyXtb(PythonPackage):
+    """Python API for the extended tight binding program package"""
+
+    homepage = "https://xtb-python.readthedocs.org"
+    url = "https://github.com/grimme-lab/xtb-python/releases/download/v22.1/xtb-22.1.tar.gz"
+
+    maintainers = [
+        *PythonPackage.maintainers,
+        "awvwgk",
+    ]
```

Comment:
I don't particularly care about the default list of maintainers being overridden. I agree it would be nice to have a way to add maintainers instead of overwriting. If you want to submit a proposal for this I would be happy to review. `maintainers += ...` doesn't work does it?

Reply:
No, `maintainers += ..` results in `NameError: name 'maintainers' is not defined`.

The above also fails (on a simple test) with a `SyntaxError` on the asterisk.  

I've been recommending the following syntax:

`maintainers = PythonPackage.maintainers + ...`

Reply:
The "correct" way to do this is inside an `__init__()` function, probably. Class inheritance doesn't define this variable in advance, but looking it up via self.maintainers works.

Probably wisest to specify the behavior via a new dedicated `extra_maintainers` or something, yeah...